requirements.txt

```diff
@@ -54,7 +54,7 @@ nbformat==5.1.3
 nest-asyncio==1.5.1
 notebook==6.4.2
 numpy==1.21.1
-nupack==4.0.0.27
+#nupack==4.0.0.27 breaks the requirements install ...
```

Comment:
No, it does not break if a path to the library is passed in the pip install command.